components/Form/Autocomplete/Autocomplete.jsx

```diff
@@ -152,26 +151,4 @@ AutocompleteComponents.propTypes = {
   }),
 };
 
-const Autocomplete = ({ name, control, ...otherProps }) => (
```

Comment:
We still need the "controlled" version of the component otherwise it's not going to work in the forms.